airbyte-integrations/connectors/source-mailchimp/source_mailchimp/manifest.yaml

```diff
@@ -0,0 +1,253 @@
+version: 0.51.2
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - lists
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path:
+        - '{{ parameters.field_path }}'
+  basic_authenticator:
+      type: BasicHttpAuthenticator
+      password: '{{ config.credentials.apikey }}'
+      username: 'anystring'
+  oauth_authenticator:
+      type: BearerAuthenticator
+      api_token: '{{ config.credentials.access_token }}'
```

Comment:
This is not implemented correct.

Reply:
It is correct as that is the implementation from the previous Python version

https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mailchimp/source_mailchimp/source.py#L44-L47

This is how the access_token value is gotten

https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mailchimp/source_mailchimp/spec.json#L75-L85

Reply:
You're looking in the wrong place
https://github.com/airbytehq/airbyte/blob/cae3ac4b715189ff3df5ef66766d62d57b1c14b8/airbyte-integrations/connectors/source-mailchimp/source_mailchimp/spec.json#L13-L42